internal/nginx/config/validation/http_filters.go

```diff
@@ -39,3 +43,60 @@ var supportedRedirectStatusCodes = map[int]struct{}{
 func (HTTPRedirectValidator) ValidateRedirectStatusCode(statusCode int) (valid bool, supportedValues []string) {
 	return validateInSupportedValues(statusCode, supportedRedirectStatusCodes)
 }
+
+func (HTTPRequestHeaderValidator) ValidateRequestHeaderName(name string) error {
+	return validateHeaderName(name)
+}
+
+var requestHeaderVariables = map[string]struct{}{
+	"request_uri":             {},
```

Comment:
being that it's already implemented, do see an issue with just leaving it in?

Reply:
one reason: some of the variables contain global NGINX information which regular users (app developer) don't have access to - since they don't deal with NGINX directly. or SSL related information (since SSL termination is managed through Gateway resource, by cluster admins).  The concern here is data exposure